.github/workflows/checks.yml

```diff
@@ -2,9 +2,6 @@ name: Run checks
 
 on: [push]
 
-permissions:
```

Comment:
This permission should stay as it's needed for other jobs. I don't _think_ that'll fix the test failure, but 🤷🏻